test/functional/combine_logs.py

```diff
@@ -78,10 +77,10 @@ def read_logs(tmp_dir):
     for each of the input log files."""
 
     # Find out what the folder is called that holds the debug.log file
-    chain = glob.glob("{}/node0/*/debug.log".format(tmp_dir))
-    if chain:
-        chain = chain[0]  # pick the first one if more than one chain was found (should never happen)
```

Comment:
Nit, not critical here but if it should never happen I would throw an exception instead

Reply:
I added an assert, as suggested above.